endercore/src/main/java/com/enderio/core/mixin/client/ItemRendererMixin.java

```diff
@@ -16,8 +16,8 @@
     @Inject(at = @At("RETURN"), method = "renderGuiItemDecorations(Lnet/minecraft/client/gui/Font;Lnet/minecraft/world/item/ItemStack;IILjava/lang/String;)V")
```

Comment:
Does this work automatically with the obf. method name?

Reply:
I'm not sure. But I think a refmap is generated to tra slate the mixins to obfuscated names

Reply:
Yeah this is mapped to obfuscated names